tests/input/example-class.php

```diff
@@ -124,6 +124,13 @@ public function trySwitchSpace() : void
         try {
             $var = 1;
             switch (self::VERSION) {
+                case 1:
```

Comment:
Let's please isolate this in another file to make things easier to understand and maintain.